src/components/SearchItem/index.tsx

```diff
@@ -13,6 +13,7 @@ export interface ISearchItem {
   displayName: string;
   categoryIds?: string[];
   avatarFileId?: string;
+  id?: string;
```

Comment:
Can we remove this ?

Reply:
I am afraid, we cannot. I understand all of targetId, userId and this id are the same. But this is used by MentionInput library. !_!. I cannot change library default props {id: number} to {targetId: number}.